### PR TITLE
Develop next release

### PR DESCRIPTION
**Pull request by Kaskyi**
Use toaster.handle to change $.notify to mat-snack-bar
```
toaster:
  handle: 'mat-snackbar'
```

### CODE REVIEW

File: packages/components/src/feedback/feedback.component.ts [hunk 1]
```diff
@@ -13,7 +14,10 @@ export interface FeedbackConfig {
 @Component({
     selector: 'xm-feedback',
     template: `
-        <button *ngIf="config" (click)="create(config.feedback.url)" mat-icon-button>
+        <button *ngIf="config"
```
Comment (Sumragen) [inline packages/components/src/feedback/feedback.component.ts:17]: `config` comes from outside and `config.feedback.url` possible could be undefined.

File: packages/xm-alert/src/toaster/xm-alert.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ export class XmAlertComponent implements OnInit {
     @Input() public dismissible: boolean;
     @Input() public type: string;
     @Input() public message: string;
-    @Output() public close: EventEmitter<any> = new EventEmitter();
+    @Output() public close: EventEmitter<void> = new EventEmitter();
```
Comment (Sumragen) [inline packages/xm-alert/src/toaster/xm-alert.ts:19]: redundant property `close`. Cannot find any usage

**Review by Sumragen — CHANGES_REQUESTED** [review]

File: packages/components/src/by-entity-id/by-entity-id-cell.component.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    public ngOnChanges(): void {
+        this.update();
+    }
+
+    public ngOnInit(): void {
+        this.update();
+    }
+
+    protected update(): void {
+        if (this.value) {
```
[the remark this answers is not in the document]
Comment (Kaskyi) [reply packages/components/src/by-entity-id/by-entity-id-cell.component.ts:46]: done

File: packages/components/src/feedback/feedback.component.ts (hunk 1 above)
Comment (Kaskyi) [reply packages/components/src/feedback/feedback.component.ts]: done

File: packages/xm-alert/src/toaster/xm-alert.ts (hunk 2 above)
Comment (Kaskyi) [reply packages/xm-alert/src/toaster/xm-alert.ts]: done

File: packages/components/src/by-entity-id/by-entity-id-cell.component.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    constructor(
+        protected factoryService: EntityCollectionFactoryService,
+    ) {
+    }
+
+    public ngOnChanges(): void {
+        this.update();
+    }
+
+    public ngOnInit(): void {
```
[the remark this answers is not in the document]
Comment (Kaskyi) [reply packages/components/src/by-entity-id/by-entity-id-cell.component.ts]: done

File: packages/components/src/by-entity-id/by-entity-id.component.ts [hunk 5]
```diff
@@ -1,61 +1,48 @@
...
     constructor(
-        private factoryService: EntityCollectionFactoryService,
+        protected factoryService: EntityCollectionFactoryService,
     ) {
+        super(factoryService);
     }
 
     public ngOnChanges(): void {
-        this.update();
+        super.ngOnChanges();
```
[the remark this answers is not in the document]
Comment (Kaskyi) [reply packages/components/src/by-entity-id/by-entity-id.component.ts]: done

**Review by Sumragen — APPROVED** [review]

**Review by EKrasovskiy — APPROVED** [review]

**Pull request merged**